public/app/features/explore/utils/links.ts

```diff
@@ -159,46 +159,51 @@ export const getFieldLinksForExplore = (options: {
     });
 
     const fieldLinks = links.map((link) => {
-      if (!link.internal) {
-        const replace: InterpolateFunction = (value, vars) =>
-          getTemplateSrv().replace(value, { ...vars, ...scopedVars });
+      let internalLinkSpecificVars: ScopedVars = {};
```

Comment:
Can you comment here what changed? The diff isn't really helpful as things were moved around.